cms/templates/widgets/header.html

```diff
@@ -36,12 +35,10 @@ <h1 class="branding"><a href="/">
             grading_url = reverse('contentstore.views.grading_handler', kwargs={'course_key_string': unicode(course_key)})
             advanced_settings_url = reverse('contentstore.views.advanced_settings_handler', kwargs={'course_key_string': unicode(course_key)})
             tabs_url = reverse('contentstore.views.tabs_handler', kwargs={'course_key_string': unicode(course_key)})
+            certificates_url = ''
```

Comment:
I can see there is a pattern here of building the URLs in the template, but I think we should instead build the certificate_url in the view logic.  Bonus points for moving the other URL builders into the view as well, but not necessary for this PR.